app/models/search.rb

```diff
@@ -74,6 +74,10 @@ def with_stories_in_domain(base, domain)
     end.where('domains.domain = ?', domain)
   end
 
+  def with_stories_pointing_to_url(base, url)
+    base.where('url LIKE ?', "#{url}%")
```

Comment:
Two questions:
1. I don't think this correctly escapes the url. What if the user enters `url:htt%`? I think this would find every story with a url. And permit SQL injection. (Maybe I'm missing it and this class escapes at a different level? There's a lot happening in here.)
2. How does this handle merged stories?

Reply:
1. ActiveRecord uses parameter binding. The data isn't interpolated directly into the string but sent separately.
2. I will have to investigate that.